src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+function getPartner({
+  isMarried,
+  sex,
+  partnerName,
+}) {
+  if (isMarried) {
+    if (sex === 'm') {
+      return `${partnerName} is my wife`;
+    }
+
+    return `${partnerName} is my husband`;
+  }
+
+  return 'I am not married';
+}
```

Comment:
No need to create separate function, it is not reused anywhere.
Better to just create variable inside the component, perform all manipulations with it, and use it in jsx.